debug_toolbar/panels/settings.py

```diff
@@ -24,10 +24,16 @@ def title(self):
         )
 
     def generate_stats(self, request, response):
+        def catch_force_errors(force_function, value):
+            try:
+                return force_function(value)
+            except DjangoUnicodeDecodeError:
+                return "Debug toolbar was unable to parse value"
```

Comment:
There are a few places we use `force_str` and I suspect we may want to swap all of them out with this new function. Perhaps we should create a `sanitize.py` file and put this in there with a doc string explaining the interface and what it does. Then use that everywhere we've been using `force_str`. I'd consider keeping the name as `force_str` too, but that's just my preference. We'd want to have an explicit test covering the functionality of this new function too.

There are some other sanitizing functions in `utils.py` that can be moved over too, though that should be a separate commit. Or I can do that.